### PR TITLE
Update runner

### PR DESCRIPTION
**Pull request by Peter9192**
Some updates to wrf-runner as I'm writing the blogpost

Updated namelists and geogrid tables to be consistent with current runs in snakemake workflow
Also added some further tweaks, e.g. `opt_output_from_geogrid_path`

### CODE REVIEW

**Comment by Peter9192** [comment]
This seems to work okayish now. Wanna have a quick look @ClaireDons?

File: wrf-runner/wrf_runner.sh [hunk 1]
```diff
@@ -12,76 +19,56 @@ set -euxo pipefail
...
 export WPS_HOME=$HOME/wrf-model/WPS
 export WRF_HOME=$HOME/wrf-model/WRF
-export WRF_RUNNER=$HOME/Urban-M4/misc/wrf-runner
 export OUTPUT_DIR=$HOME/Urban-M4/experiments
 export DATA_HOME=/projects/0/prjs0914/wrf-data/default
 
-# Define experiment name and if w2w should be run (lcz setups only)
+# Set path to executables
+export WPS_HOME=$HOME/wrf-model/WPS
+export WRF_HOME=$HOME/wrf-model/WRF
```
Comment (ClaireDons) [inline wrf-runner/wrf_runner.sh:33]: These 2 lines seem to be a duplicate of the lines above.

File: wrf-runner/wrf_runner.sh [hunk 2]
```diff
@@ -12,76 +19,56 @@ set -euxo pipefail
...
-ln -sf $WRF_HOME/run/CAMtr_volume_mixing_ratio.RCP8.5 CAMtr_volume_mixing_ratio
-ln -sf $WRF_HOME/run/ozone* $RUNDIR
-ln -sf $WRF_HOME/run/RRTMG* $RUNDIR
-ln -sf $WRF_HOME/run/*.TBL $RUNDIR
-
-# Run WRF
-f90nml $WRF_RUNNER/namelist.input namelist.input
-$WRF_HOME/run/real.exe
-$WRF_HOME/run/wrf.exe
+$WRF_HOME/real.exe
```
Comment (ClaireDons) [inline wrf-runner/wrf_runner.sh:68]: Not sure if your setup is now different but I had to change this to `$WRF_HOME/run/real.exe` for it to work

File: wrf-runner/wrf_runner.sh [hunk 3]
```diff
@@ -12,76 +19,56 @@ set -euxo pipefail
...
 
-# Report status
-status=$? && [ $status -eq 0 ] && echo "Run successful"
+# Configure OpenMP thread affinity
+# export OMP_NUM_THREADS=${SLURM_CPUS_PER_TASK}
+# export OMP_PLACES=cores
+# export OMP_PROC_BIND=close
+# srun $WRF_HOME/wrf.exe
```
Comment (ClaireDons) [inline wrf-runner/wrf_runner.sh:74]: Same as above, I had to change the path to include 'run'

**Review by ClaireDons — APPROVED** [review]
Hey, this works nicely and is a lot neater than before! Just a couple of comments about paths that I needed to change so that it would run on my end. 

I also noticed that if you have an error and you need to re-run that you have to delete the experiment directory each time. I guess that is one of the downsides of not using snakemake.

**Pull request merged**